.github/workflows/predict.yml

```diff
@@ -18,109 +18,62 @@ on:
-  # If placing the script in the individual repo, find and replace '${{ inputs.model-id }}' with '$MODEL_ID'
-  # MODEL_ID: ${{ github.event.repository.name }}
-  # Replace ${{ inputs.SHA }} with ${SHA} in the individual model repo case.
-  # SHA: ${{ github.sha }}
+      # numerator:
+      #   required: true
+      #   type: number
+      # denominator:
+      #   required: true
+      #   type: number
+      # local-filename:
+      #   required: true
+      #   type: string
+      # output-path:
+      #   required: true
+      #   type: string
+      # model-id:
+      #   required: true
+      #   type: string
+      # SHA:
+      #   required: true
+      #   type: string
 
 jobs:
   infer-and-upload:
     runs-on: ubuntu-latest
     steps:
       - name: Print system details
         run: sudo lshw -short
-      
-      - name: Check into ersilia-os/ersilia repo 
-        uses: actions/checkout@v3
-        with:
-          repository: ersilia-os/ersilia
-
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@master
-        with:
-           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
-           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-           aws-region: eu-central-1
 
-      - name: Get input library
-        run: |
-          if [[ ${{ inputs.sample-only }} ]]; then
-            echo "Getting subset of reference library"
-            filename="reference_library_${{ inputs.sample-only }}.csv"
-          else
-            echo "Getting whole reference library"
-            filename="reference_library.csv"
-          fi
-          aws s3 cp s3://precalculations-bucket/"$filename" $filename
+      - name: checkout repo content
+        uses: actions/checkout@v4
 
-      - name: Split library by partition variables
+      - name: Delete unnecessary folders
         run: |
-          numerator=${{ inputs.numerator }}
-          denominator=${{ inputs.denominator }}
-          if [[ ${{ inputs.sample-only }} ]]; then
-            filename="reference_library_${{ inputs.sample-only }}.csv"
-          else
-            filename="reference_library.csv"
-          fi
-          
-          ## Total # lines in given reference library
-          total_lines=$(wc -l < "$filename")
-          echo $total_lines
-          
-          # Determine the # of file lines per partition
-          ((lines_per_file = (total_lines + $denominator-1) / $denominator))
-
-          # Split the file
-          split -l $lines_per_file -a 4 -d --additional-suffix=.csv "$filename" partition_
-          
-          # Add 'smiles' to header of all partition files beyond the first
-          for (( i=1; i<$denominator; i++ ))
-          do
-            partition=$(printf "partition_%04d.csv" $i)
-            sed -i '1i smiles' "$partition"
-          done
-
-          # List files in dir
-          ls
+          shopt -s extglob
+          rm -rf !(scripts)
```

Comment:
shouldn't need to do this, plus we want to be able to use the module we're writing

Reply:
https://github.com/ersilia-os/model-inference-pipeline/pull/29/commits/577990f483ef2d8c1d4cd17637514ef170fc31f0